tests/ActivityLoggerTest.php

```diff
@@ -195,6 +195,40 @@ public function it_will_use_the_logged_in_user_as_the_causer_by_default()
         $this->assertEquals($userId, $this->getLastActivity()->causer->id);
     }
 
+    /**
+     * @test
+     *
+     * @requires !Travis
+     */
+    public function it_can_log_activity_using_an_anonymous_causer()
+    {
+        activity()
+            ->causedByAnonymous()
+            ->log('hello poetsvrouwman');
+
+        $this->assertEquals(null, $this->getLastActivity()->causer_id);
+        $this->assertEquals(null, $this->getLastActivity()->causer_type);
+    }
+
+    /**
+     * @test
+     *
+     * @requires !Travis
+     */
+    public function it_will_override_the_logged_in_user_as_the_causer_when_an_anonymous_causer_is_specified()
```

Comment:
Thanks for this test! It was one of my fears that we have to update the initial causer setter. But good that it works. :)